docs/src/index.md

```diff
@@ -54,6 +54,19 @@ or other graphical user interface. It is primarily a programming library, not a
 user-facing application. However, it could be used as the backend for such an
 application.
 
+### What is a GAT?
+Generalized Algebraic Theories (GATs) are the backbone of Catlab so let's expand a bit on GATs and how they fit into the bigger picture of algebra.
+An algebraic structure, like a group or category, is a mathematical object whose axioms all take the form of equations that are universally quantified (the equations have no exceptions). That’s not a formal definition but it’s a good heuristic. There are different ways to make this precise. The oldest, going back to universal algebra in the early 20th centrury, are algebraic theories.
+
+[Universal Algebra](https://en.wikipedia.org/wiki/Universal_algebra) (sometimes called general algebra) is the field of mathematics that studies algebraic structures themselves, not examples ("models") of algebraic structures.
+For instance, rather than take particular groups as the object of study, in universal algebra one takes the class of groups as an object of study. In an algebraic theory, you have a collection of (total) operations and they obey a set of equational axioms. Classically, there is only a single generating type, but there are also typed or multi-sorted versions of algebraic theories. Most of the classical structures of abstract algebra, such as groups, rings, and modules, can be defined as algebraic theories.
+
+Importantly, a category cannot be defined as a algebraic theory. The reason is that the operation of composition is partial, since you can only compose morphisms with compatible (co)domains. Now, categories sure feel like algebraic structures, so people have come up with generalizations of algebraic theories that accomodate categories and related structures.
```

Comment:
> Importantly, a category cannot be defined as a algebraic theory.

Is this known? It might be better to phrase this as "Importantly, there is no known way to define a category as algebraic theory." (or similar). You can often encode a surprising amount of structure into simpler systems (c.f. the simply-typed lambda calculus and the typed SK-calculus). It's not entirely implausible that some dependent algebraic structures could be encoded as (multisorted) algebraic structures.

Reply:
Since this is not a research article, I think this is fine. I'll defer to @epatters for his judgement.

Reply:
I'm pretty sure this is true. I know I've seen it claimed in various places. For example, in their introduction to [this paper](https://www.cs.bham.ac.uk/~sjv/partialalgebras.pdf), Vickers and Palmgren state that "the theory of categories is not algebraic", by which they mean that it cannot be construed as a multi-sorted algebraic theory. Unfortunately they don't give a proof or a reference. The way you would prove such a thing is to show that any category of models of an algebraic theory has a certain property that the category of categories fails to possess. For example, this is how you prove that the [theory of fields is not algebraic](https://mathoverflow.net/questions/3003/in-what-sense-are-fields-an-algebraic-theory) (!).

Anyway, if you find a good reference on this, please let us know.

Reply:
I looked around for a reference, but couldn't find one, so I asked on mathoverflow and received [an elegant explanation](https://mathoverflow.net/a/354926/152679).

Reply:
Very nice! I'll update the docs with a link to this answer.